app/collections/components/AdminEditors.tsx

```diff
@@ -60,6 +67,23 @@ const EditorCard = ({ editor, isAdmin, isSelf, refetchFn }) => {
             </select>
             <SetEditorToInactiveModal editor={editor} refetchFn={refetchFn} />
             <DeleteEditorModal editor={editor} refetchFn={refetchFn} />
+            <Modal
+              title="Confirm changing your role to a user"
+              body={
+                <span>
+                  Are you sure you want to remove yourself as {currentRole.toLowerCase()} of this
+                  collection? By doing so, you will lose access to the admin panel.
+                </span>
+              }
+              primaryAction="Assign yourself as a user"
```

Comment:
```suggestion
              primaryAction="Change Role"
```

This is more consistent with the rest of the copy.